### PR TITLE
Tests for sensitive terminology.

### PR DESCRIPTION
**Pull request by squeedee**
Introduce inclusivity terms checks.

### CODE REVIEW

File: hack/inclusive.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+trap cleanup EXIT
+
+mkdir -p ./temp-inclusiveness
+
+echo "# downloading sensitive terms"
+curl https://s3.amazonaws.com/srp-cli/api-rules/regex > ./temp-inclusiveness/regex
+
+echo "# selecting files to scan"
+find . \( -name "*.go" -o -name "*.yal" -o -name "*.md" -o -name "*.sh" \) | grep -vE "/node_modules/|/target/|/dist/" > ./temp-inclusiveness/files
```
Comment (cirocosta) [inline hack/inclusive.sh]:
```
find . \( -name "*.go" -o -name "*.yal" -o -name "*
                                    ^
```

did you mean `yaml`?

Comment (squeedee) [reply hack/inclusive.sh]: I did,  good catch.

Comment (squeedee) [inline hack/inclusive.sh]:
```suggestion
find . \( -name "*.go" -o -name "*.yaml" -o -name "*.md" -o -name "*.sh" \) | grep -vE "/node_modules/|/target/|/dist/" > ./temp-inclusiveness/files
```

**Review by cirocosta — APPROVED** [review]
lgtm!

**Pull request merged**